resources/views/PageDesign/Partials/Header/DefaultHeader.twig

```diff
@@ -1,6 +1,6 @@
 {% import "Ceres::PageDesign.Macros.LayoutContainer" as LayoutContainer %}
 
-<header id="page-header" class="default-header d-print-none">
+<header id="page-header" class="default-header d-print-none sticky-top">
```

Comment:
```suggestion
{% set lowClsHeader = config("Ceres.header.low_cls_mode") %}

<header id="page-header" class="default-header d-print-none {% if lowClsHeader == "true" %}sticky-top{% endif %}">
```

**chore**: Add the same condition as for the ShopBuilder header. Now that we've added the config, we might as well use it consistently across both templates.